src/components/UI/Loader/styles.ts

```diff
@@ -0,0 +1,8 @@
+import { MantineTheme, DEFAULT_THEME } from "@mantine/core";
+import { theme } from '@/theme';
+
+const colors = { ...DEFAULT_THEME.colors, ...theme.colors } as Record<string, readonly string[]>;
```

Comment:
OK so in here you can do `const colors =  theme.colors`  and it should work, right now it will not work because this branch in behind main branch, merge main into this branch first